src/main/java/echo/Deadline.java

```diff
@@ -0,0 +1,27 @@
+package echo;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+public class Deadline extends Task {
+
+    protected LocalDate by;
```

Comment:
Would it be better if the variable can be named to be more descriptive like 'byDate'